### PR TITLE
index code fix

### PR DESCRIPTION
**Pull request by JulianS61**
just some minor change mainly to index

### CODE REVIEW

File: app/views/equipment/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1>Equipment List</h1>
+<h1>Offers List</h1>
```
Comment (tieuu) [inline app/views/equipment/index.html.erb:1]: I think keep it as "Equipment" instead of "Offers" sounds more user friendly~

**Review by tieuu — CHANGES_REQUESTED** [review]
Hi i think overall is fine, personally i prefer "Equipment" instead of "Offers" in the index page, what do you guys think @VincentMsi @danpring ?

File: app/views/equipment/index.html.erb (hunk 1 above)
Comment (VincentMsi) [reply app/views/equipment/index.html.erb:1]: I have no preference. Equipment sounds more technical, offers sounds more marketplace

Comment (VincentMsi) [reply app/views/equipment/index.html.erb:1]: Sorry I merged before I saw your comment @tieuu

**Pull request merged**